juniper_official/System/system-cpu.rule

```diff
@@ -69,6 +69,7 @@ healthbot {
                 sensor components {
                     where "/components/component/properties/property/@name == 'cpu-utilization-idle'";
                     path /components/component/properties/property/state/value;
+                    zero-suppression;
```

Comment:
If we want to zero-suppress cpu-utilization-idle, can we first declare it as type int?

Reply:
Corrected